templates/wireguard_conf.epp

```diff
@@ -46,10 +46,11 @@ MTU=<%= $mtu %>
 
 <% $peers.each |$peer| { -%>
 
+[Peer]
 <% if $peer['description'] { -%>
-# <%= $peer['description'] %>
+# friendly_name = <%= $peer['description'] %>
+Description=
```

Comment:
I don't think Description is a valid field in wireguard configuration files, no?